{{cookiecutter.project_slug}}/merge_production_dotenvs_in_dotenv.py

```diff
@@ -0,0 +1,69 @@
+import os
+from typing import Sequence
+
+import pytest
+
+ROOT_DIR_PATH = os.path.dirname(os.path.realpath(__file__))
+PRODUCTION_DOTENVS_DIR_PATH = os.path.join(ROOT_DIR_PATH, '.envs', '.production')
+PRODUCTION_DOTENV_FILE_PATHS = [
+    os.path.join(PRODUCTION_DOTENVS_DIR_PATH, '.django'),
+    os.path.join(PRODUCTION_DOTENVS_DIR_PATH, '.postgres'),
+    os.path.join(PRODUCTION_DOTENVS_DIR_PATH, '.caddy'),
+]
+DOTENV_FILE_PATH = os.path.join(ROOT_DIR_PATH, '.env')
```

Comment:
It looks like this file is Python 3 only? If so I'd recommend using [Pathlib](https://docs.python.org/3/library/pathlib.html), it has a nicer API to work with paths IMO.

Reply:
Thanks for pointing this out! That's the 'issue' with almost the entire Cookicutter Django. To my mind, we should start a separate PR covering Pathlib migration.